makefiles/kconfig.mk

```
@@ -140,6 +143,17 @@ $(KCONFIG_GENERATED_DEPENDENCIES): FORCE | $(GENERATED_DIR)
 	      printf "config %s\n\tbool\n\tdefault y\n", toupper($$0)}' \
 	  | $(LAZYSPONGE) $(LAZYSPONGE_FLAGS) $@
 
+# All directories in EXTERNAL_MODULES_DIR which have a Kconfig file
+EXTERNAL_MODULE_DIRS_WITH_CONFIG ?= $(sort $(foreach dir,$(EXTERNAL_MODULE_DIRS),\
+                                    $(dir $(wildcard $(dir)/Kconfig))))
```

Any special reason to keep the directory list instead of the file list?
```suggestion
EXTERNAL_MODULE_KCONFIGS ?= $(sort $(foreach dir,$(EXTERNAL_MODULE_DIRS),\
                                    $(wildcard $(dir)/Kconfig)))
```

---

No, that is a leftover of a previous solution I had, will address

---

Addressed as suggested.